src/dropdown.ts

```diff
@@ -2326,7 +2325,9 @@ export default class Dropdown extends LitElement implements FormControl {
   #onInputResize() {
     if (this.#inputElementRef.value) {
       this.isInputOverflowing =
-        this.#inputElementRef.value.scrollWidth >
+        // One is subtracted to account for an apparent Chrome bug where `scrollWidth`
+        // is off by one relative to `clientWidth` when they should be the same.
+        this.#inputElementRef.value.scrollWidth - 1 >
```

Comment:
The fix. I was subtracting one [selectively](https://github.com/CrowdStrike/glide-core/blob/main/src/dropdown.ts#L1792-L1798) before because I only observed the bug in a few situations. Now I'm doing it everywhere.